web/gds-user-ui/src/components/Addresses/index.tsx

```diff
@@ -1,12 +1,11 @@
-import { Heading, Text } from '@chakra-ui/react';
+import { Heading, Stack, Text } from '@chakra-ui/react';
 import { Trans } from '@lingui/react';
-import FormLayout from 'layouts/FormLayout';
 import React from 'react';
 import AddressList from './AddressList';
 
 const Addresses: React.FC = () => {
   return (
-    <FormLayout>
```

Comment:
The `FormLayout` component has a styles set to display the border that we wanted to remove from around the form's smaller sections. The `FormLayout` component is used throughout the UI, so replacing it with the `Stack` component allows for the content to be grouped together. This change will also make it easier to apply specific styles to a smaller section within a form.